ddl/tests/adminpause/BUILD.bazel

```diff
@@ -29,7 +29,6 @@ go_test(
     ],
     embed = [":adminpause"],
     flaky = True,
-    race = "on",
```

Comment:
why delete `race` detector?

Reply:
> why delete `race` detector?

Consider the CPU & Memory & Time consuming.
And it's safe to delete it, since we have other tests can ensure it.